feat(CFW): protection rule supports to clear the rule hit count (#4456)
huaweicloud/terraform-provider-huaweicloud: docs/resources/cfw_protection_rule.md

  + **0**: inbound;
  + **1**: outbound;

> * `rule_hit_count` - (Optional, String) The number of times the protection rule is hit.
>   Setting the value to 0 can serve to clear it.

Reviewer: ```suggestion
* `rule_hit_count` - (Optional, String) The number of times the protection rule is hit.
  Setting the value to **0** will clear the hit count. Value options: **0**.
```

Author: fixed
